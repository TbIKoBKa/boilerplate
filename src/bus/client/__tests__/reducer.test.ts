/**
 * @jest-environment jsdom
 */

// Reducer
import { togglersReducer as togglers } from '../togglers';

// Actions
import { togglerCreatorAction } from '../togglers';

describe('togglers actions', () => {
    test('should handle set toggler action', () => {
        expect(togglers(void 0, togglerCreatorAction({ type: 'isOnline', value: false }))).toMatchSnapshot();
    });
});
