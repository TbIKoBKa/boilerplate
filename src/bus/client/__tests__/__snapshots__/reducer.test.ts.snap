// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`togglers actions should handle set toggler action 1`] = `
Object {
  "isFilmsFetching": true,
  "isOnline": false,
  "isPeopleFetching": true,
  "isTodosFetching": true,
  "isUsersFetching": true,
}
`;
